README.md

```diff
@@ -72,14 +72,25 @@ subspace-node-x86_64-*-snapshot --chain testnet --wasm-execution compiled --exec
 subspace-farmer-x86_64-*-snapshot farm
 ```
 
-**Important Notes!**
+### Important Notes!
 
+***Identity Management***: If you would like to import your polkadot.js wallet so you can interact with the network, you may do so via 
+>You may need to wipe the farm with the `wipe` command if you ran the farmer prior to importing a mnemonic. 
```

Comment:
I think this line should go after, it looks strange. And make sure to have white space after `>` so it is easier to read.

Reply:
You mean line 77 below line 78?

Reply:
No, the whole note after the command. Having that sentence after `via` looks weird.